src/components/pages/PasswordReset/PasswordReset.js

```diff
@@ -0,0 +1,86 @@
+import React from 'react';
+import { toast } from 'react-toastify';
+import { useState } from 'react';
+
+import validateResetPassword from './resetPasswordValidation';
+import PasswordResetForm from './PasswordResetForm';
+import axiosWithHeader from '../../../utils/axiosWithHeaders';
+import { pathObj } from '../../../utils/generalVariables';
+
+const PasswordReset = props => {
+  const [state, updateState] = useState({
+    newPassword: '',
+    confirmNewPassword: '',
+    oldPassword: '',
```

Comment:
If this is not doing anything, I think we can remove it. I thought we will be sending it to the backend to check if it is correct (if someone forgot to logout and someone else came in and tried to change the password), but if we're not using it we can remove it